tests/plugins/table/_helpers/headersCorrectness.js

```diff
@@ -0,0 +1,22 @@
+/* exported assertHeadersCorrectnesssAfterManipulation */
+
+function assertHeadersCorrectnesssAfterManipulation( input, expected ) {
+	'use strict';
+
+	return function() {
+		var bot = bender.editorBots.editor;
+		bot.setHtmlWithSelection( bender.tools.getValueAsHtml( input ).replace( ';', '' ) );
```

Comment:
Ok, I see it fails if you remove another `replace` from line `16`. There is a dedicated method for comparing HTML which normalizes it before comparison, `assert.isInnerHtmlMatching( ... )` or the one you have used `assert.beautified.html`.

So when setting HTML `.replace( ';', '' )` is not needed because browser will format it in it's own way. And for asserting you can use `assert.beautified.html` as now but you need to pass proper configuration so styles are normalized (see [benderjs/benderjs-yui-beautified](https://github.com/benderjs/benderjs-yui-beautified/blob/a3ca835bb06c6dca6241c02112f21c1b55daa1f0/lib/client.js#L81) and [_benderjs/ckeditor/static/tools.js](https://github.com/ckeditor/ckeditor-dev/blob/c6969047a007e10d9bb2907ba11d0f25de1295d9/tests/_benderjs/ckeditor/static/tools.js#L286)).

Reply:
That is one cool API, thanks!